### ISSUE TITLE
ASMA: some (new?) instructions aren't supported

### ISSUE THREAD

**Issue by Fish-Git** [s390guy/SATK#11]
I am currently using: &nbsp;<b>(*)</b>

&nbsp; &nbsp; &nbsp; **Revision: 7565b3e8ac73858f80f55e7068772558e89d70d9**
&nbsp; &nbsp; &nbsp; Date: **3/24/2017** 11:07:51 AM
&nbsp; &nbsp; &nbsp; Message: **Fix font inconsistency in macro doc headings in func.odt and func.pdf**

The following should (hopefully) be self-explanatory:

```
00002310  A7FA 0001                                282          AHI   R15,1            increment failure count
                                                   283          CIJNL R15,6,SUCCESS    give up after 6 attempts (expected)
          ** [283] @[283]-1 macro definition for CIJNL not found in MACLIB path as CIJNL.mac
```

I realize the difficulty involved in trying to keep your product up to date with IBM's latest developments! We have the same problem with Hercules! Right? IBM does things faster than we can keep up!

No rush on this issue BTW. It's easy enough to work around. I just thought it should be reported.

Thanks for an otherwise _fantastic_ product, Harold!

_**SATK/ASMA totally rocks!**_ &nbsp; `:-D`
___
<b>(*)</b>&nbsp; _See issue #9 the for reason why._

**Comment by s390guy** [comment]
Yes, I am behind.  Trying to get back into things.  3/24/17 is not that behind.  I have not made many commits in a couple of years.

**Comment by s390guy** [comment]
This specific extended mnemonic is part of the recent commits of extended mnemonics.  In particular this extended mnemonic, CIJNL should now be supported.

Please review doc/asma/Instruction_Status.pdf or doc/asma/Instruction_Status.odt for any other extended mnemonics that you found missing.
